src/webots/vrml/WbProtoManager.cpp

```diff
@@ -52,8 +52,6 @@ WbProtoManager::WbProtoManager() {
   mTreeRoot = NULL;
   mExternProtoCutBuffer = NULL;
 
-  mImportedFromSupervisor = false;
-
```

Comment:
This cause #5078.
It should be re-added otherwise the `mImportedFromSupervisor` is not initialized.

Reply:
It seems it was introduced by mistake when merging with origin/master.
in any case it is not related to the fix.